include/class_info.php

```diff
@@ -50,105 +52,97 @@ public static function import_status() {
 
 		$arrIndexed = self::number_of_entries();
 
-		if(count($arrPostCount) > 0)
-		{
-			$countIndexed = 0;
-
-			foreach($arrPostCount as $posts)
-			{
-				if($posts->pinged == "indexed" || $posts->pinged == "linksconverted")
-				{
-					$countIndexed = $posts->entryCount;
-				}
-				else
-				{
-
-					$countImported = $posts->entryCount;
-				}
-			}
+		if(count($arrPostCount) == 0)
+			return "No entries have been imported yet. <a href=\"" . $_SERVER['REQUEST_URI']  . "\">refresh page</a>";
```

Comment:
Moved this condition from the bottom of the function to here in order to reduce nesting.